builder/bundles.go

```diff
@@ -323,11 +326,14 @@ func queryRpmName(packageCmd []string, pkgName string, repo string) string {
 	queryStringRpm = append(queryStringRpm, pkgName)
 	outBuf, _ := helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})
 	if outBuf.String() != "" {
-		rpm := strings.Split(outBuf.String(), "\n")
-		_, file := filepath.Split(rpm[0])
-		return file
+		out := strings.Split(outBuf.String(), "\n")
+		pURL, err := url.Parse(out[0])
```

Comment:
if url is not an file scheme, location will give url of the upstream repo but not the path to the rpms stored locally.
for example location will give https://cdn.download.clearlinux.org/releases/31930/clear/x86_64/os/Packages/filesystem-3.0.14-200.x86_64.rpm
we instead need the downloadDir as the full path

Reply:
good catch.. I was making the assumption that queryRpmFullPath() will be called only for file based scheme. Will fix.